src/entities/campaign/components/CampaignCarouselItem.tsx

```
@@ -72,7 +72,25 @@ export const CampaignCarouselItem = ({ data }: { data: Campaign }) => {
           <p className="mt-4 text-start md:h-28">
-            {data?.description ? truncate(data.description, 100) : ""}
+            <div
+              className="prose prose-sm max-w-none overflow-hidden"
+              style={{
+                display: "-webkit-box",
+                WebkitLineClamp: 3,
+                WebkitBoxOrient: "vertical",
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+              }}
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)